routes/reviewRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+const upload = require("../util/multer");
+const accessToken = require("../middleware/auth")
+
+const { reviewController } = require("../controllers")
+
+router.post("/", upload.single("image"), accessToken, reviewController.postReview);
+router.patch("/", upload.single("image"), accessToken, reviewController.editReview);
+router.delete("/", reviewController.deleteReview);
```

Comment:
RESTful 한 엔드포인트를 작성하는데 있어서 지양해야할 몇가지 중 하나는 uri 끝에 '/'를 포함하지 않는 것입니다!

추가로 review를 delete를 하는데 있어서도 인가과정에 대한 미들웨어가  적용 되어야 하지 않을까요?
```suggestion
router.post("", upload.single("image"), accessToken, reviewController.postReview);
router.patch("", upload.single("image"), accessToken, reviewController.editReview);
router.delete("", reviewController.deleteReview);
```